python/equalizer.cpp

```diff
@@ -0,0 +1,15 @@
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
+#include <vector>
+
+#include <stempy/image.h>
+#include <stempy/equalizer.h>
+
+namespace py = pybind11;
+
+using namespace stempy;
+
+PYBIND11_MODULE(_image, m)
```

Comment:
This pybind11 module probably needs to be named something other than `_image`, as that name is already taken. It also looks like there might be several header files included that are not needed.

Reply:
We can probably just add this to the _image module, rather than having a separate module.

Reply:
@psavery I actually haven't really got into the pybind11 step yet. It was just there but it is not finished yet. 
@cjh1 Sure, I will look into a way to add it to the _image module, thank you for your suggestion.